text/0044-secret-zeroize-drop.md

```diff
@@ -0,0 +1,99 @@
++ Feature name: `secret-zeroize-drop`
++ Start date: 2020-07-08
++ RFC PR: [iotaledger/bee-rfcs#44](https://github.com/iotaledger/bee-rfcs/pull/44)
++ Bee issue: [iotaledger/bee#104](https://github.com/iotaledger/bee/issues/104)
+
+# Summary
+
+This RFC does not really introduce a feature but rather a recommendation when dealing with sensitive data; their memory
+should be securely erased when not used anymore.
+
+# Motivation
+
+When dealing with sensitive data like key materials (e.g. seeds, private keys, ...), it is wise to explicitly clear
+their memory to avoid having them continue existing and potentially exposing them to an attack.
+
+Since such an attack already assumes unauthorized access to the data, this technique is merely an exploit mitigation by
+ensuring sensitive data are no longer available.
```

Comment:
**is** no longer available

Reply:
Well technically data is a plural form (plural of datum)

Reply:
yup, technically you're right. Found [this article](https://www.theguardian.com/news/datablog/2010/jul/16/data-plural-singular) about this topic. Seems like english says to Latin: Go f.... yourself :trollface:

Reply:
But I'm not english :trollface:

Reply:
Jokes aside, just by reading it both are fine, I feel like I prefer the plural one though.

Reply:
alrighty 👍

Reply:
Unless @zesterer has a more formal opinion of course

Reply:
Done.